### PR TITLE
Frontdoor with waf

### PR DESCRIPTION
**Pull request by jyotsnaravikumar**
## Purpose
Basic Terraform for provisioning Frontdoor with WAF

## Does this introduce a breaking change?
[ ] Yes
[X] No

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

```
az login
az account set -s <subscription_id>
https://github.com/Azure-Samples/frontdoor-appservice-vnet-terraform.git
cd frontdoor-appservice-vnet-terraform
git checkout frontdoor-with-waf
terraform init
terraform validate
terraform plan
terraform apply

```

* Test the code
* Verify in the Azure Portal Frontdoor is created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,75 @@
...
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
+``` shell
+az login
+az account set -s <subscription_id>
 
+terraform init
+terraform validate
+terraform plan
```
Comment (edaena) [inline README.md]: plan and apply need to use the terraform.tfvars file `terraform apply -var-file="terraform.tfvars"`

File: README.md [hunk 2]
```diff
@@ -1,57 +1,75 @@
...
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
+This project provides a sample terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing app service, routing rules with  caching config exposed on a private vnet
```
Comment (edaena) [inline README.md]: app service -> Azure App Service

**Review by edaena — CHANGES_REQUESTED** [review]
There are a couple of suggestions in the documentation. Other than that, it looks good.

File: README.md (hunk 1 above)
Comment (jyotsnaravikumar) [reply README.md]: Done!

**Review by yradsmikham — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jyotsnaravikumar) [reply README.md]: Done!

**Pull request merged**